### PR TITLE
✨ Create Menu cards

### PR DESCRIPTION
**Pull request by phlgr**
Introducing the layout system:
https://getbootstrap.com/docs/5.0/layout/grid/#auto-layout-columns

Components used in this PR:
https://getbootstrap.com/docs/5.0/components/card/

Happy Hacking 👾

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,10 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (lmachens) [inline src/index.html:5]: Why did you change this?

**Review by lmachens — COMMENTED** [review]
good job 🚀

File: src/index.html (hunk 1 above)
Comment (phlgr) [reply src/index.html:5]: `shrink-to-fit=no` was only ever used in a small portion of iOS and Safari, it is no longer necessary as of today. 
HTML5 Boilerplate and other popular repos also removed it. https://www.scottohara.me/blog/2018/12/11/shrink-to-fit.html
